build-gluster-org/scripts/user-update.py

```diff
@@ -0,0 +1,114 @@
+import requests
+import re
+import os
+import subprocess
+import json
+
+#Collects the gluster-maintainers names
+def create_user_data(repo_path, username, password):
+    os.chdir(repo_path)
+    user_data = requests.get('https://api.github.com/orgs/gluster/teams/gluster-maintainers/members', auth=(username,password)).json()
+    return user_data
+
+#Function which creates the dev path
+def create_new_dev_branch(branch_name):
+    os.system("git checkout master")
+    os.system("git pull origin master")
+    os.system("git checkout -b {}".format(branch_name))
+
+#Function to find out the names which are missing in the jenkins conf file
+def search_pat(repo_path, filename, string_to_search,user_data):
+    os.chdir(repo_path)
+    INDENTATION_VALUE = ""
+    NOT_FOUND = []
+    for i, line in enumerate(open(filename)):
+        if string_to_search in line:
+            INDENTATION_VALUE=re.match(r"\s*", line).group()
+    for username in user_data:
+        with open(filename) as f:
+            if re.search(username['login'], f.read()):
+                continue
+            else:
+                NOT_FOUND.append("{}  ".format(INDENTATION_VALUE)+"- "+username['login'])
+    f.close()
+    return NOT_FOUND
+
+#Function which appends the names which are not present in the jenkins conf file
+def edit_file(repo_path,filename,pattern,NOT_FOUND):
+    os.chdir(repo_path)
+    with open(filename, "r") as input_file:
+        buffer = input_file.readlines()
+
+    with open(filename, "w") as output_file:
+        search_pattern = re.compile(r'\s*%s'%pattern) 
+        for line in buffer:
+            if search_pattern.search(line):  
+                for names in NOT_FOUND:
+                    line = line + str(names) + "\n"
+            output_file.write(line)
+
+    input_file.close()
+    output_file.close()
+
+#Checks the files which are modified
+def check_untracked_files(repo_path,pat,branch_name):
+    os.chdir(repo_path)
+    FILES = []
+    os.system('git status')
+    out = subprocess.Popen(['git', 'status'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout, stderr = out.communicate()
+    output_decoded = stdout.decode('UTF-8').split('\n')
```

Comment:
So, is the code python 2 or python 3 ? I think that our server is too old for py3 :/